tools/cockpit.spec

```diff
@@ -52,6 +52,10 @@ BuildRequires: docbook-style-xsl
 BuildRequires: keyutils-libs-devel
 BuildRequires: dbus-devel
 BuildRequires: glib-networking
+
+# There is a segfault in g_source_add_child_source that was fixed in
+# 2.37.3-2-g1d5c815
+BuildRequires: glib2-devel >= 2.40
```

Comment:
IIUIC this spec change is necessary to automatically get yum to pull in the updated glibc2.

because, even if glibc2 is available as an update, but the user just installs cockpit, currently the updated glibc2 will not be instaled/updated.